group_vars/all

```diff
@@ -60,7 +60,8 @@ base_packages:
     - coreutils
     - moreutils
     - pass
-    - reflector
+    - packer # instead of aura-bin
```

Comment:
Same as above for packer, and no reflector... 'mirror.archlinuxarm.org' uses Geo-IP.